vpr/src/base/vpr_types.h

```diff
@@ -1412,6 +1412,40 @@ struct t_det_routing_arch {
     /// the CUSTOM switch block type. See comment at top of SRC/route/build_switchblocks.c
     std::vector<t_switchblock_inf> switchblocks;
 
+    /// Sub type and Fs are applied to pass tracks
+    int sub_fs;
+
+    /// Subtype of switch blocks.
+    enum e_switch_block_type switch_block_subtype;
+
+    // Following options are used only for tileable routing architecture
+
+    /// Whether the routing architecture is tileable
+    bool tileable;
+
+    /// Allow connection blocks to appear around the perimeter programmable block (mainly I/Os)
+    bool perimeter_cb;
+
+    /// Remove all the routing wires in empty regions
+    bool shrink_boundary;
+
+    /// Allow routing channels to pass through multi-width and multi-height programmable blocks.
+    bool through_channel;
```

Comment:
This seems to control a similar thing but in a different way as the regular rr-graph generator's switchblock location pattern (https://docs.verilogtorouting.org/en/latest/arch/reference/#tag-%3Cswitchblock_locationspattern=).  Makes it extra important to document that this is only for the tileable rr-graph.